sections/all_map_locations.tex

```diff
@@ -310,8 +311,8 @@ \subsection*{Other fields}
 \end{figure}
 \vfill
 
-\begin{figure*}[!hb]
-  \centering
-  \includegraphics[width=\linewidth]{\art/gold_dragon.jpg}
-\end{figure*}
+\begin{scaledfigure}[blanker]
+	\centering
+	\includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/gold_dragon.jpg}
```

Comment:
Please use spaces instead of tabs.
```suggestion
  \centering
  \includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/gold_dragon.jpg}
```